### ISSUE TITLE
[Accessibility]A11y_VSCode_VSCLicenseAgreement_Screenreader : Screenreader does not read the control type of the links

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51842]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
JAWS 18.0.5038 , NVDA 2018.2, MAS1.3.1

**Repro Steps:**
1.On VSC Setup Wizard, click on "Next" button.
2.Check whether ScreenReader reads control type of the links or not.

**Actual:**
Screenreader does not read the control type of the links.

**Expected:**
Screenreader should read the control type of the links

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to know  the controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes

**Attachment for Reference:**
![accessibility a11y_vscode_vsclicenseagreement_screenreader](https://user-images.githubusercontent.com/39204909/41396205-c6b31742-6fcd-11e8-9e9d-dade9307c121.png)

**Comment by joaomoreno** [comment]
@AccessibilityTestingTeam-TCS This needs to go into https://github.com/jrsoftware/issrc
